prow/plugins/label/label.go

```diff
@@ -107,7 +114,30 @@ func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) err
 		if err := gc.AddLabel(owner, repo, number, sigLabel); err != nil {
 			log.WithError(err).Errorf("Github failed to add the following label: %s", sigLabel)
 		}
+	}
+
+	labelsToRemove := []string{}
+	for _, match := range removeLabelMatches {
+		for _, removeLabel := range strings.Split(match[0], " ")[1:] {
```

Comment:
it would be really nice if we could reuse this stuff, since right now the code "select the labels to be added" and "select the labels to be removed" is duplicated.